src/main/resources/LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright {yyyy} {name of copyright owner}
+   Copyright 2016-2019 SparklingComet @ http://shanerx.org & KillerOfPie @ https://killerofpie.github.io/
```

Comment:
I am changing the url before the release

Reply:
cool